circle.yml

```diff
@@ -28,7 +28,7 @@ test:
     - cd $IMPORTPATH && scripts/gofmt-check.sh
     - cd $IMPORTPATH && scripts/end2end_test.sh
     - cd $IMPORTPATH && scripts/tools_gitignore.sh
-    - cd $IMPORTPATH && go test -v -run='^$' -bench=. $(go list ./... | grep -v /vendor/)
+    - cd $IMPORTPATH && go test -v -run='^$' $(go list ./... | grep -v /vendor/)
```

Comment:
this command now runs no tests and no benchmarks, it's essentially just a compilation check.
this can be removed since we already unit test (and compilation check) earlier.
in fact, i have removed it in master, so maybe just rebase it.

Reply:
ok, rebased :+1: